src/ts/src/data/cellRenderers.tsx

```diff
@@ -44,17 +43,11 @@ const TREE_CUSTOM_RENDERERS: Record<string | number, CellRenderer> = {
   },
   name: (value: JSONPrimitive): JSX.Element => {
     const stringValue = stringGuard(value);
+
     const treeID = stringValue.split(" ")[0];
-    const treeLocation = `${stripProtocol(
-      process.env.API_URL
-    )}/api/auspice/view/${treeID}`;
+
     return (
-      <Modal
-        data={createTreeModalInfo(
-          `https://nextstrain.org/fetch/${treeLocation}`
-        )}
-        className={style.cell}
-      >
+      <Modal data={createTreeModalInfo(treeID)} className={style.cell}>
```

Comment:
We don't need to construct the URL here, now that it's the new `ConfirmButton`'s job!